Source/Flow/Private/AddOns/FlowNodeAddOn.cpp

```diff
@@ -68,7 +68,14 @@ bool UFlowNodeAddOn::IsSupportedInputPinName(const FName& PinName) const
 		return true;
 	}
 
-	return FindFlowPinByName(PinName, InputPins) ? true : false;
+	if (const FFlowPin* FoundFlowPin = FindFlowPinByName(PinName, InputPins))
+	{
+		return true;
```

Comment:
Broke this back out into if/else so we can set breakpoints.

Reply:
Why not do it like this?
```
bool bFoundPin = FindFlowPinByName(PinName, InputPins) != nullptr;
return bFoundPin;
```
Or this? 
```
bool bFoundPin = !!FindFlowPinByName(PinName, InputPins);
return bFoundPin;
```

Reply:
Those are more efficient than the ternary approach.

But none of them allow you to set a breakpoint on the return statement that is unique for the 'return true' separate from the 'return false' case.

It's something I like to have when I'm debugging (being able to set a breakpoint on either the return true or return false case, in isolation).  But if we don't expect to need to do that, then your first option is the preferred way in my estimation.

Reply:
You can always set conditional breakpoint to achieve the same behaviour
![image](https://github.com/MothCocoon/FlowGraph/assets/48297221/3cab342d-a37c-4ac7-910f-66797619ad58)

Reply:
It's faster to set a regular breakpoint.

I'm not particularly concerned either way.
---
The standard I have set for my engineers is to prefer if/else over ternary for anything that we would ever want to set a breakpoint on either branch result (that is, only use ternary for trivial cases like null-protection, or setting a constant variable, etc.)

Otherwise, we suggest avoiding a branches when possible.

I saw this ternary as something that should be shifted in one way or the other:  either it should be an if/else, or it should be branchless bool return.

That's my reasoning, and why I reverted the code back to if/else.  I prefer the breakpoint option that if/else gives over ternary.